package.json

```diff
@@ -185,6 +185,8 @@
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
       "\\.(css|less|styl)$": "identity-obj-proxy"
     },
+    "verbose": true,
+    "testURL": "http://localhost/",
```

Comment:
Why do we need these props?

Reply:
When trying to run `npm run jest` [this error](https://github.com/jsdom/jsdom/issues/2304)  came up and I followed the suggestions in the [comments](https://github.com/jsdom/jsdom/issues/2304#issuecomment-440642168) in order to fix it. Do you wish me to change it back?

Reply:
Somehow I can now run the test without adding those lines. @anasasilva can you re-run the test without those lines?

Reply:
I did and they didn't work but I deleted them anyway